Clarify Body package data algorithm with multipart/form-data MIME type

eehakkin opened the pull request:
Fixes #392.

annevk commented:
@youennf what do you think? This seems like an improvement. I have a couple nits I'll push in a commit once everyone is on board with this.

yutakahirano commented:
lg

annevk commented:
@youenff ping.

annevk commented:
@youennf ping.

youennf commented:
Sorry for the delay, looks like an improvement to me as well!

I would split some sentences in two so that they are more readable.
entry might need a link reference.
And I guess entry has a "value" and not "values"

eehakkin commented:
I split some sentences and corrected plurality errors.
Bikeshed correctly links <a>entry</a> to https://xhr.spec.whatwg.org/#concept-formdata-entry.
Don't know what happens with Travis.

annevk commented:
I think you can satisfy Travis by rebasing at this point. Sorry about that.

annevk commented on fetch.bs:
```diff
@@ -4363,10 +4363,30 @@ runs the associated steps:
+    This is done regardless of the presence or the value of a `<code>Content-Type</code>` header
+    and regardless of the presence or the value of a `<code>charset</code>` parameter.</p>
+
+    <p>A part whose `<code>Content-Disposition</code>` header contains a `<code>name</code>`
```
I think this would be clearer as a note (class=note) since it's just a statement of fact.

eehakkin replied:
Done.

annevk commented on fetch.bs:
```diff
@@ -4363,10 +4363,30 @@ runs the associated steps:
+
+    <p>Each part whose `<code>Content-Disposition</code>` header does not contain
+    a `<code>filename</code>` parameter must be parsed into an <a>entry</a>
+    whose value is a {{USVString}} whose content is the <a>utf-8</a> decoded content of the part.
```
Use "whose value is the <a>UTF-8 decoded</a> content of the part" instead.

eehakkin replied:
Done.

annevk commented:
@eehakkin are you okay with the changes I made? Looks pretty good now.

eehakkin commented:
@annevk: Yes, sure. I am okay with the changes you made.

annevk commented:
Great, thanks!
